### DISCUSSION TITLE
draw columns to fill whole height based on max Y value in visible chart.

### DISCUSSION THREAD

**Discussion by slickeer** [patrykandpatrick/vico#760] [Questions] [answered]
### Question

Hello,

is there a way to draw columns to fill whole height based on max Y in visible chart while scrolling ?
Also drawing max value on startAxis on the left for current visible items. 

https://github.com/patrykandpatrick/vico/assets/11398162/5039514d-ced9-46ab-bc98-3537b41d0041

Thank you.

### Vico version(s)

v2.0.0-alpha.21

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment, marked as answer]
Hello, please see #403.
